python/rainbow/simulators/prox_soft_bodies/api.py

```diff
@@ -85,6 +87,14 @@ def create_soft_body(engine, body_name, V, T) -> None:
     body.offset = engine.number_of_nodes
     engine.number_of_nodes += len(body.x0)
 
+    # setting up the hash grid
+    engine.hash_grid.set_hash_table_size(len(body.surface), False)
+ 
+    if engine.hash_grid.cell_size == 0:
+        engine.hash_grid.cell_size = HashGird.compute_optial_cell_size(body.x0, body.surface)
+    else :
+        engine.hash_grid.cell_size = (HashGird.compute_optial_cell_size(body.x0, body.surface)+engine.hash_grid.cell_size)/2 
```

Comment:
So if bvh is used then hash grid are still set up? Should it not only be initialized if this is the algorithm of choice?